<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>language_switcher</class>
 <widget class="QDialog" name="language_switcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Choose your Language</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">Chose your language:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="font">
      <font/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="translator_link">
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string notr="true">I also want to translate Voc2brain</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font/>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
